### ISSUE TITLE
Feature Request: Save file to a specific folder in Google Drive and other cloud storages

### ISSUE THREAD

**Issue by tobiaskarch** [keeweb/keeweb#933]
When I create a new file and save it to Google Drive, the file is saved in `/`. I have a folder structure in GDrive and would like to save the file in a folder of my choice (e.g. `/data/KeeWeb/`).

**Comment by antelle** [comment]
The suggestion is to move it manually. Uploading to Google Drive, Dropbox, etc... inside a folder could be very complicated because of shares and other stuff, I'd say, it's not our priority.

**Comment by tobiaskarch** [comment]
Yes, moving is a solution when using the `https://www.googleapis.com/auth/drive` scope (used by KeeWeb, access to all files). I experimented with my own Google Drive OAuth2 Client-ID and the `https://www.googleapis.com/auth/drive.file` scope (Per-file access to files created or opened by the app).

But there are two problems with this:

1. If I move the file created by KeeWeb, KeeWeb no longer has access to the file.
2. To open an existing file with KeeWeb via GDrive, you need to specify a URL of the KeeWeb instance that KeeWeb can use to open the GDrive file.

I currently use my own Client-ID with the `drive` scope due to the two problems with `drive.file` which is absolutely no problem. Nevertheless, I hope that this feature request will eventually come.
